### ISSUE TITLE
Update URLs to use https if possible

### ISSUE THREAD

**Issue by sbmlsecretary** [sbmlteam/sbml-specifications#355]
Various URLs in the spec can be updated to use https instead of http.  From Matthias:

Hi Lucian, 
I just had a look at the spec. Yes, namespaces must stay, but many links can be updated. Here my impression:

Replace
http://www.biopax.org/ -> https://www.biopax.org/ (url, https exists)
http://www.uniprot.org/ -> https://www.uniprot.org/
http://identifiers.org/ -> https://identifiers.org/
http://www.ebi.ac.uk/ -> https://www.ebi.ac.uk/
http://www.genome.jp/dbget-bin/www_bget?ec:3.5.4.4 -> https://www.genome.jp/dbget-bin/www_bget?ec:3.5.4.4
http://www.cellml.org/specification -> https://www.cellml.org/specification
http://tools.ietf.org/html/rfc6350 -> https://tools.ietf.org/html/rfc6350

Remove
http://www.godatabase.org/cgi-bin/amigo/go.cgi?view=details&query=GO:0007268 (link is dead, resource probably does not exist any more)
http://us.expasy.org/cgi-bin/nicezyme.pl?3.5.4.4 (link is dead, resource does not exist any more)
http://www.math.uwaterloo.ca/bingalls/Pubs/conservation.pdf (link is dead, resource does not exist anymore)
.
Not sure about
http://biomodels.net/sbo/ -> https://www.ebi.ac.uk/sbo/main/ (forwards, https exists, careful this is also part of namespaces, only replace when the website is mentioned)

Don't replace
http://sbml.org/ (urls, no https yet)
http://www.sbml.org/sbml/ (namespace)
http://www.w3.org/ (namespace)
http://www.mysim.org/ns (namespace)
http://www.struct.org/ns (namespace)
http://www.othersim.com/ (namespace)
http://www.sbgn.org/ (url, no https yet)
http://www.psidev.info/index.php?q=node/60 (url, no https yet)
http://biomodels.net/ (used in definitionURL)
http://co.mbine.org/ (no https yet, part of namespaces)
http://purl.org/dc/ (namespace)
http://www.informatics.jax.org/searches/GO.cgi?id=GO:0007268 (url, no https yet)
http://dublincore.org/* (url, no https yet)

Reported by: luciansmith

Original Ticket: [sbml/sbml-specifications//357](https://sourceforge.net/p/sbml/sbml-specifications//357)

**Comment by sbmlsecretary** [comment]
- **labels**:  --> Level 3 Version 1 Core, Level 3 Version 2 Core


Original comment by: luciansmith

**Comment by sbmlsecretary** [comment]
My inclination is to not change the biomodels.net/sbo URL since it's used in namespaces, to be consistent.  Thanks for the list!

Original comment by: luciansmith

**Comment by sbmlsecretary** [comment]
- **status**: open --> pending
- **Group**: Reported-Proposed --> Accept-formatting-and-typos


Original comment by: luciansmith

**Comment by sbmlsecretary** [comment]
This is now fixed in SVN, including updating the bibliography to get some of the other URLs you found--thanks!  Otherwise, my list matched yours--got new URLs for genome.jp examples.

I was not able to mark any updated URL in red except when it was not flagged as a URL (as in the examples in the 'annotation' section), but I don't think this is a huge problem.

Given that the editors already signed off on this change in the other issue tracker item (https://sourceforge.net/p/sbml/sbml-specifications/293/) for l2v5, I'm marking this as accepted/pending as well, though if any editor wants to chime in for any reason, feel free.

Also added to http://sbml.org/Documents/Specifications/SBML_Level_3/Version_2/Core/Confirmed_issues_in_the_Level_3_Version_2_Core_Specification .

Original comment by: luciansmith

**Comment by sbmlsecretary** [comment]
- **labels**: Level 3 Version 1 Core, Level 3 Version 2 Core --> Level 3 Version 2 Core, Level 3 Version 1 Core Release 2


Original comment by: luciansmith

**Comment by sbmlsecretary** [comment]
- **status**: pending --> closed


Original comment by: luciansmith

**Comment by sbmlsecretary** [comment]
Released.

Original comment by: luciansmith
